packages.json

```diff
@@ -1,6 +1,12 @@
 {
   "packages": {
-    "plugins": [
+      "plugins": [
+    {
+        "name": "FindIt",
+        "url": "https://github.com/rogermolas/findit-for-xcode",
+        "description": "Plug-in for browsing stackoverflow website inside XCode IDE.",
+        "screenshot": "https://github.com/rogermolas/findit-for-xcode/blob/master/demo.gif"
```

Comment:
It won't work to link to the github UI, we need to link to the image directly. Could you update this to:

    https://raw.githubusercontent.com/rogermolas/findit-for-xcode/master/demo.gif

Thanks!